crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -222,7 +222,12 @@ impl LogManagerActor {
         let path = match event {
             FsWatchEvent::Modified(path) => path,
             FsWatchEvent::FileDeleted(path) => path,
-            FsWatchEvent::FileCreated(path) => path,
+            // if we created the file ealier and now modify it, keep only later modify. some editors
```

Comment:
```suggestion
            // if we created the file earlier and now modify it, keep only later modify. some editors
```

Reply:
I must've copied this comment on autopilot, because only now I realize that actually it doesn't talk about why `FileCreated` is disabled, but about something else. My bad. Will replace that with some more helpful comment.